Description: Handle clazy plugin.
 - Make analyze tool use clazy checks.
 - Use the correct clazy plugin name. It was changed in clazy 1.5.
 .
 This patch is a merge from two different patches done by Lisandro.
Author: Alexander Volkov <a.volkov@rubistech.ru>
  Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Reviewed-by: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Last-Update: 2020-06-16
---
 src/plugins/clangtools/clangtidyclazyrunner.cpp |    1 +
 1 file changed, 1 insertion(+)

--- a/src/plugins/clangtools/clangtidyclazyrunner.cpp
+++ b/src/plugins/clangtools/clangtidyclazyrunner.cpp
@@ -67,7 +67,7 @@ static QStringList clazyPluginArguments(
     QStringList arguments;
 
     if (diagnosticConfig.isClazyEnabled()) {
-        arguments << XclangArgs({"-add-plugin", "clazy"});
+        arguments << XclangArgs({"-add-plugin", "clazy", "-load", "ClazyPlugin.so"});
         if (!diagnosticConfig.clazyChecks().isEmpty())
             arguments << XclangArgs({"-plugin-arg-clazy", diagnosticConfig.clazyChecks()});
     }
